scripts/linters/custom_eslint_checks/rules/e2e-practices.js

```diff
@@ -134,7 +134,7 @@ module.exports = {
 
     var checkElementSelectorProtractor = function(node) {
       var thirdPartySelectorPrefixes = (
-        ['.modal', '.select2', '.CodeMirror', '.toast', '.ng-joyride', '.mat']);
+        ['.modal', '.select2', '.CodeMirror', '.toast', '.mat']);
```

Comment:
Should `.ng-joyride` be replaced with `.joyride`?

Reply:
done